app/javascript/onboarding/components/ProfileForm.jsx

```diff
@@ -22,7 +22,7 @@ export class ProfileForm extends Component {
     this.user = userData();
     this.state = {
       groups: [],
-      formValues: {},
+      formValues: { user: { username: this.user.username } },
```

Comment:
refactor (blocking): It needs to be just a `username` property in the component state, not `user.username`.

Below you can see the initial state in `user.username`, but when you change the username, it's populated under just `username` in the component state.

![image](https://user-images.githubusercontent.com/833231/108940129-d173f280-7620-11eb-8b17-70bccdac4210.png)

Reply:
:upside_down_face: